frontend/src/global_styles/layout/work_packages/_table_baseline.sass

```diff
@@ -56,6 +56,11 @@
     // Ensure items are not taking full width
     align-items: flex-start
 
+    &.startDate
+      padding-left: $work-package--start-date-display-field-padding-left
+
+  .spot-icon
+    vertical-align: middle
```

Comment:
Out of curiosity: Why did you add this rule? It doesn't make any difference for me locally. 🤔

Reply:
Hey @HDinger I think it is due to a merge from outdated dev branch. I'll remove it.